.github/workflows/release.yml

```diff
@@ -0,0 +1,68 @@
+name: Release
```

Comment:
This builds all the containers needed for a Faasm release. It'll be slow, but it's a start